### PR TITLE
Update getting started on macos

### LINKED ISSUE #1648: Update tutorials/using-on-macos.md for MSBuild/csproj
Getting started with .NET Core on macOS, using Visual Studio Code
* https://docs.microsoft.com/en-us/dotnet/articles/core/tutorials/using-on-macos
* https://github.com/dotnet/docs/blob/master/docs/core/tutorials/using-on-macos.md

xref https://github.com/dotnet/docs/issues/1574 (includes `dotnet new` commands)

### PR DESCRIPTION
**Pull request by BillWagner**
update the getting started on Mac topic for the csproj build system

Targeting the csproj branch

/cc @mairaw 

Fixes #1648

### CODE REVIEW

File: docs/core/tutorials/using-on-macos.md [hunk 1]
```diff
@@ -161,34 +140,37 @@ about in the [Learn C#](../../csharp/index.md) section.
...
+</ItemGroup>
 ```
 
 The `library` node specifies that this dependency should resolve to a project
 in the current workspace. Without explicitly specifying this, it's possible
 that the test project would build against a NuGet package of the same name.
 
 Now that the dependencies have been properly configured, let's create
-the tests for your library. Open `Tests.cs` and
+the tests for your library. Open `UnitTest1.cs` and
```
Comment (mairaw) [inline docs/core/tutorials/using-on-macos.md]: nit: there are still some files styled with backticks versus italic

File: docs/core/tutorials/using-on-macos.md [hunk 2]
```diff
@@ -207,31 +189,27 @@ namespace TestApp
...
+run the tests.
 The xUnit console test runner will run the one test, and report
 that it is passing. 
 
 ### Writing the console app
 
-In your terminal, cd to the `golden/src/app` directory. Run `dotnet new`
-to create a new console application.
+In your terminal run `dotnet new console -o app`
+to create a new console application. This proejct is also
```
Comment (mairaw) [inline docs/core/tutorials/using-on-macos.md]: typo: proejct

File: docs/core/tutorials/using-on-macos.md [hunk 3]
```diff
@@ -161,34 +140,37 @@ about in the [Learn C#](../../csharp/index.md) section.
...
-    "target": "project"
-  }
-}
+above. The `dotnet add reference` command does that:
+
+```
+dotnet add test-library/test-library.csproj reference library/library.csproj
+```
+
+Or, you can manually edit the test-library.csproj file and add the
```
Comment (mairaw) [inline docs/core/tutorials/using-on-macos.md]: style *test-library.csproj*

**Review by mairaw — COMMENTED** [review]
Looks good overall. Left a couple of comments (some are nits) for you to address. In the future, I think we could review this topic to be less verbose with its steps, but something to be looked at later. Thanks!

File: docs/core/tutorials/using-on-macos.md [hunk 4]
```diff
@@ -29,7 +24,7 @@ it points out the differences for the Windows platform.
 ## Prerequisites
 
 Before starting, you'll need to install the [.NET Core SDK](https://www.microsoft.com/net/core),
-currently in a preview release. The .NET Core SDK includes the latest release
+currently in RC4 pre-release. The .NET Core SDK includes the latest release
```
Comment (BillWagner) [inline docs/core/tutorials/using-on-macos.md]: This one is still open after the last commit. Awaiting @leecow for the correct version.

File: docs/core/tutorials/using-on-macos.md [hunk 5]
```diff
@@ -29,7 +24,7 @@ it points out the differences for the Windows platform.
...
 of the .NET Core framework and runtime.
 
 You'll also need to install [Visual Studio Code](http://code.visualstudio.com).
```
Comment (BillWagner) [inline docs/core/tutorials/using-on-macos.md]: Fixed.

File: docs/core/tutorials/using-on-macos.md [hunk 6]
```diff
@@ -49,95 +44,79 @@ terminal window, if you prefer).
...
-your development experience more productive.
+use of the library. 
 
 Let's start by creating those folders. In the terminal, create a 'golden'
-directory. Under that directory create `src` and `test`
-directories. Under `src` create `app` and `library` directories. In `test`
-create a `test-library` directory. You can do this either using the terminal
-in VS code, or by clicking on the parent folder in VS Code and selecting the
-"New Folder" icon.
+directory. In VS Code, open the *golden* directory. This directory is the root of your solution. Run the `dotnet new` command to create a new solution:
```
Comment (BillWagner) [inline docs/core/tutorials/using-on-macos.md]: Fixed.

File: docs/core/tutorials/using-on-macos.md [hunk 7]
```diff
@@ -49,95 +44,79 @@ terminal window, if you prefer).
...
+cd to *golden/* and type the command:
 
-At this point, your directory tree should look like this:
+```
+dotnet new classlib library
+```
 
+This creates a library project, with two files: *library.csproj* and
+*Class1.cs* in the *library* directory. You want that library project
+included in your solution. Run the `dotnet sln` command to add the newly
```
Comment (BillWagner) [inline docs/core/tutorials/using-on-macos.md]: Fixed. Note: This now must wait until the preview 3 folder merges with the tools folder.

File: docs/core/tutorials/using-on-macos.md (hunk 3 above)
Comment (BillWagner) [inline docs/core/tutorials/using-on-macos.md]: Fixed.

File: docs/core/tutorials/using-on-macos.md (hunk 1 above)
Comment (BillWagner) [inline docs/core/tutorials/using-on-macos.md]: Fixed.

File: docs/core/tutorials/using-on-macos.md (hunk 2 above)
Comment (BillWagner) [inline docs/core/tutorials/using-on-macos.md]: Fixed.

File: docs/core/tutorials/using-on-macos.md (hunk 4 above)
Comment (mairaw) [inline docs/core/tutorials/using-on-macos.md]: Let's just leave as .NET Core SDK right now with no version indication and we can fix that later when we know more @BillWagner

**Review by mairaw — APPROVED** [review]

**Pull request merged**